builder/dockerfile/internals.go

```diff
@@ -298,3 +340,39 @@ func hostConfigFromOptions(options *types.ImageBuildOptions) *container.HostConf
 		ExtraHosts: options.ExtraHosts,
 	}
 }
+
+func containsWildcards(name, platform string) bool {
+	for i := 0; i < len(name); i++ {
+		ch := name[i]
+		if ch == '\\' && platform != "windows" {
+			i++
+		} else if ch == '*' || ch == '?' || ch == '[' {
+			return true
+		}
+	}
+	return false
+}
+
+func validateCopySourcePath(imageSource *imageMount, origPath, platform string) error {
+	// validate windows paths from other images
+	if imageSource == nil || platform != "windows" {
```

Comment:
There's quite a few `platform string` that was already here. I think it's from @jhowardmsft's first LCOW PR. He's on a platform API for the builder, so do you want the change now or in his PR? @jhowardmsft: Thoughts?